src/DynamoCoreWpf/PublicAPI.Unshipped.txt

```diff
@@ -2435,6 +2435,8 @@ Dynamo.ViewModels.NodeViewModel.IsDisplayingLabels.set -> void
 Dynamo.ViewModels.NodeViewModel.IsExperimental.get -> bool
 Dynamo.ViewModels.NodeViewModel.IsFrozen.get -> bool
 Dynamo.ViewModels.NodeViewModel.IsFrozen.set -> void
+Dynamo.ViewModels.NodeViewModel.IsTransient.get -> bool
```

Comment:
Is this something that we actually want to be public ? I think it's ok for now and we can think/debate later on this subject.

Reply:
Good point. Let me see if I can make it internal for now

Reply:
Looks like not, the property is bind to UI layer and needs to be public, at least NodeViewModel layer, I can make the NodeModel layer internal

Reply:
If I remember right we can have something public and change our mind a little later while in master ? Or no